{{ cookiecutter.package_name }}/.readthedocs.yml

```diff
@@ -1,5 +1,25 @@
-conda:
-  file: .rtd-environment.yml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+version: 2
 
+# Set "fail_on_warning: true" for RTD to fail on warnings.
+sphinx:
+  builder: html
+  configuration: docs/conf.py
+  fail_on_warning: false
+
+# Set the version of Python and requirements required to build your docs
+# NOTE: This does not follow {{ cookiecutter.minimum_python_version }}
```

Comment:
I am not sure if one should build doc from *minimum* Python version, so I just hardcode it to Python 3.7.

Reply:
well, this is just a template, maintainers are expected to look under the hood, at least a little bit, and then your comment clears up the situation.